src/mixins/dom.js

```diff
@@ -3,17 +3,28 @@
 
 import Backbone from 'backbone';
 
+function getEl(el, context) {
+  return el instanceof Backbone.$ ? el : Backbone.$(el, context);
```

Comment:
eh... now that you pointed that out they should be separated out to `getEl(el)` and `findEl(context, el)` where `findEl will use `return getEl(context).find(el)`. Lumping them, currently, looks to be producing goofy fallbacks.